scripts/delete_branch.sh

```diff
@@ -1,44 +1,64 @@
 #!/bin/bash
 
-BRANCHES=$(ls   /var/www/vhosts/mf-chsdi3/private/branch/)
+TARGET=$1
+BRANCH=$2
+
 BASE=/var/www/vhosts/mf-chsdi3
+BRANCH_DIR=$BASE/private/branch
+CONF="$BASE/conf/00-$BRANCH.conf"
+BASEDIR="$BRANCH_DIR/$BRANCH"
+
+# Read target ips
+source <(grep "$TARGET =" deploy/deploy.cfg | sed 's/ *//g')
+IFS=', ' read -r -a TARGET_IPS <<< "${!TARGET}"
 
+if [ ${#TARGET_IPS[@]} == 0 ]; then
+    TARGET_IPS=('')
+fi
 
-if [ "$#" -ne 1 ]; then
-        echo "No branch name given"
-        echo "Possible values:"
-        for b in ${BRANCHES};
-            do
-               echo ${b}
-            done
-        exit 0
+if [ $# -ne 2 ]; then
+    echo "No branch name given, use parameter BRANCH_TO_DELETE=your_branch_name"
```

Comment:
Message is misleading: yo want two parameters

Reply:
And I would also test the number of args at the very top

Reply:
I can't do it at the very top I need all the previous variables to display the list of branches available